entity-management-web/src/main/java/eu/europeana/entitymanagement/batch/processor/EntityUpdateProcessor.java

```diff
@@ -40,15 +41,15 @@ public EntityUpdateProcessor(EntityRecordService entityRecordService,
     public EntityRecord process(@NonNull EntityRecord entityRecord) throws EuropeanaApiException {
         //TODO: Validate entity metadata from Proxy Data Source
         emEntityFieldCleaner.cleanAndNormalize(entityRecord.getExternalProxy().getEntity());
-	entityRecordService.mergeEntity(entityRecord);
+        entityRecordService.mergeEntity(entityRecord);
```

Comment:
the minimal validation must not be integrated into the dereferencing step, but as own functionality in the EntityUpdateProcessor see step 4 in the specs https://docs.google.com/document/d/16k9PcCMFwl2LXjnnzotZRPc-QqM-Ar1D0VELHt4t_hA/edit#

Reply:
Improved.